browser/components/MarkdownEditor.js

```diff
@@ -142,6 +142,12 @@ class MarkdownEditor extends React.Component {
     this.renderPreview(this.props.value)
   }
 
+  handleKeyDown(e) {
+    if (this.state.status === 'CODE' && e.key == 'Escape') {
```

Comment:
Is Focus notes the same as toggle main? Or is the escape from editing markdown? If the escape from editing makdown, I think esc is the best.